UNRELEASED.md

```diff
@@ -2,6 +2,8 @@
 
 > Place your changes below this line.
 
+**Fixed:**
+ - Cards and Panels now have a corner radius of `bpk-spacing-md` by default.
```

Comment:
```suggestion
 - The default corner radius of cards and panels has been increased from `bpkSpacingXs` to `bpkSpacingMd` (4 to 12).
```

I think this is a bit more informative as it tells them it's increased, and from what to what.